packages/code-du-travail-api/src/server/routes/modeles/index.js

```diff
@@ -2,7 +2,7 @@ const Router = require("koa-router");
 const API_BASE_URL = require("../v1.prefix");
 
 const elasticsearchClient = require("../../conf/elasticsearch.js");
-const getModeleBody = require("./searchModele.elastic");
```

Comment:
the rename is not very "search“ effective with all the file named search. I wold prefer name it getModeleBody.elastic or getMailTemplate.elastic